### PR TITLE
[SMALLFIX] Add docker file for Alluxio with fuse enabled

### PR DESCRIPTION
**Pull request by bf8086**

### CODE REVIEW

**Comment by bf8086** [comment]
@yupeng9 PTAL. Thanks!

File: integration/docker/Dockerfile.fuse [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+FROM ubuntu:16.04
+
+ARG ALLUXIO_TARBALL=http://downloads.alluxio.org/downloads/files/1.6.1/alluxio-1.6.1-bin.tar.gz
```
Comment (yupeng9) [inline integration/docker/Dockerfile.fuse]: lets update the tarball to 1.7 after the release

File: integration/docker/README.md [hunk 2]
```diff
@@ -24,3 +26,38 @@ docker run -e ALLUXIO_MASTER_HOSTNAME=ec2-203-0-113-25.compute-1.amazonaws.com a
 Additional configuration files can be included when building the image by adding them to the
 `integration/docker/conf/` directory. All contents of this directory will be
 copied to `/opt/alluxio/conf`.
+
+
+## Building docker image with fuse support
```
Comment (yupeng9) [inline integration/docker/README.md]: fuse -> FUSE

File: integration/docker/README.md [hunk 3]
```diff
@@ -24,3 +26,38 @@ docker run -e ALLUXIO_MASTER_HOSTNAME=ec2-203-0-113-25.compute-1.amazonaws.com a
...
+To build a docker image with fuse support, run
```
Comment (yupeng9) [inline integration/docker/README.md]: Can you also add a link to the FUSE doc page?

File: integration/docker/README.md [hunk 4]
```diff
@@ -24,3 +26,38 @@ docker run -e ALLUXIO_MASTER_HOSTNAME=ec2-203-0-113-25.compute-1.amazonaws.com a
...
+
+```bash
+docker build -f Dockerfile.fuse -t alluxio-fuse .
+```
+
+You can add the same arguments supported by the non-fuse docker file.
+
+
+## Running docker image with fuse support
+There are a couple extra arguments required to run the docker image with fuse support, for example:
```
Comment (yupeng9) [inline integration/docker/README.md]: could you also add the security implications of the extra parameter `SYS_ADMIN` or give a link?

File: integration/docker/README.md [hunk 5]
```diff
@@ -24,3 +26,38 @@ docker run -e ALLUXIO_MASTER_HOSTNAME=ec2-203-0-113-25.compute-1.amazonaws.com a
...
+
+```bash
+docker run -e ALLUXIO_MASTER_HOSTNAME=alluxio-master --cap-add SYS_ADMIN --device /dev/fuse  alluxio-fuse [master|worker|proxy]
+```
+
+## Extending docker image with applications
+You can easily extend the docker image to include applications to run on top of Alluxio.
```
Comment (yupeng9) [inline integration/docker/README.md]: Could you emphasize that applications and compute frameworks must be in the same image to use FUSE?

File: integration/docker/Dockerfile.fuse (hunk 1 above)
Comment (bf8086) [reply integration/docker/Dockerfile.fuse]: Done.

File: integration/docker/README.md (hunk 2 above)
Comment (bf8086) [reply integration/docker/README.md]: Done.

File: integration/docker/README.md (hunk 3 above)
Comment (bf8086) [reply integration/docker/README.md]: Done.

File: integration/docker/README.md (hunk 4 above)
Comment (bf8086) [reply integration/docker/README.md]: Done.

File: integration/docker/README.md (hunk 5 above)
Comment (bf8086) [reply integration/docker/README.md]: Done.

**Comment by yupeng9** [comment]
@bf8086 could you also update the [documentation](https://github.com/Alluxio/alluxio/blob/master/docs/en/Running-Alluxio-On-Docker.md)?

**Pull request merged**